app/api/routes.py

```diff
@@ -0,0 +1,45 @@
+from flask import current_app as app , request , abort, Blueprint
+from flask_bcrypt import generate_password_hash
+from controllers.auth import *
+from utils.decorators import admin_required , admin_manager_required
+
+mongo = app.extensions['mongo']
+role_bp = Blueprint('role',__name__)
+
+#POST Route to add manager
+@role_bp.route('/user/manager', methods=["POST"])
+@admin_required
+def add_manager():
+    if not request.json:
+        abort(500)
+    email = request.json.get("email",None)
+    password = request.json.get("password",None)
+    validate_email_pass(email,password)
+    manager = get_user_by_email(email)
+    if not manager:
+        password = generate_password_hash(password).decode('utf8')
```

Comment:
You can do this stuff (Code inside if block) inside controller instead of doing here in routes